app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/EmailVerificationConnector.scala

```diff
@@ -17,22 +17,22 @@
 package uk.gov.hmrc.cgtpropertydisposalsfrontend.connectors
 
 import cats.data.EitherT
-import configs.syntax._
 import com.google.inject.{ImplementedBy, Inject, Singleton}
 import configs.Configs
+import configs.syntax._
 import play.api.Configuration
 import play.api.libs.json.{Format, Json}
 import play.api.mvc.Call
 import uk.gov.hmrc.cgtpropertydisposalsfrontend.connectors.EmailVerificationConnectorImpl.EmailVerificationRequest
 import uk.gov.hmrc.cgtpropertydisposalsfrontend.http.HttpClient._
-import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.email.Email
-import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.name.ContactName
 import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.Error
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.name.ContactName
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.onboarding.email.Email
```

Comment:
emails are more general than onboarding - they can go at the same level as `models.name`, i.e. `models.email`

Reply:
fixed